### PR TITLE
On-/offboarding copy updates

### LINKED ISSUE #38960: [Enhancement] On-/offboarding copy updates
We've reviewed and updated the copy for the onboarding flows. Almost all copy has changed, so please look at the design file and update the text. I've marked the updated screens with a red circle.

Let's also use this opportunity to swap one of the illustrations in the feature tour for a fixed version.

**Design**
lj4AUMJnpMtUe7XqyvOONq-fi-1334_116372

`lj4AUMJnpMtUe7XqyvOONq-fi-1334-116372&mode=design`

**Acceptance criteria**

- [x] Copy in the welcome modal, the feature tour, and feedback modals matches the designs.
- [x] There's an updated illustration in the first step of the feature tour: [Tour1.zip](https://github.com/woocommerce/woocommerce/files/11881015/Tour1.zip)

### PR DESCRIPTION
**Pull request by mdperez86**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #38960

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

New users flow. Note that for this flow the store should not has any published product.
1. Make sure the `New product editor` feature is active under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure `woocommerce_block_product_tour_shown` is set to `no` and `woocommerce_product_editor_show_feedback_bar` is set to `yes` under `/wp-admin/tools.php?page=woocommerce-admin-test-helper` so `WCA Test Helper` should be installed
3. Visit `/wp-admin/admin.php?page=wc-admin&path=/add-product` you should see the `welcome modal` with title `Meet the product editing form` and description `Discover the product form's unique features with a quick overview of what's included.`
4. After clicking `View highlights` button you should see a tour modal with title `Fresh and modern interface`, description `Everything you need to create and sell your products, all in one place. From photos and descriptions to pricing and inventory, all of your product settings can be found here.  `  and the illustration shared in second AC from #38960
5. Clicking the `Next` button a second modal should be shown with title `Content-rich product descriptions` and description `Show off what's great about your products and engage your customers with content-rich product descriptions. Add images, videos, and any other content they might need to make a purchase.`
6. Clicking the `Next` button a third modal should be shown with title `Lightning fast performance` and description `Get your products listed and available for purchase in no time! Our modern technology ensures a reliable and streamlined experience.`
7. Clicking the `Next` button a four modal should be shown with title `More features are on the way` and description `We're actively working on adding more features to the product form, including the ability to add digital products, variations, and more. Watch this space!`
8. After closing the last modal a bottom bar should slides out from the bottom of the screen and with some background blur. Check the design from #38960
9. Clicking `Share feedback` link from the bottom bar should show a feedback modal with tile `What do you think of the new product form?`
10. When clicking `turn it off` link from the bottom bar should show a new modal with the following checkboxes 
image and the primary button should be `Send`

Old users flow. Note that for this flow the store should has at least one published product.
1. Make sure the `New product editor` feature is active under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure `woocommerce_block_product_tour_shown` is set to `no` and `woocommerce_product_editor_show_feedback_bar` is set to `yes` under `/wp-admin/tools.php?page=woocommerce-admin-test-helper` so `WCA Test Helper` should be installed
3. Visit `/wp-admin/admin.php?page=wc-admin&path=/add-product` you should see the `welcome modal` with title `Welcome to the new product form!` and description `Discover its new features and improvements with a quick overview of what's included.`

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement

#### Comment 

</details>

### CODE REVIEW

File: packages/js/customer-effort-score/changelog/enhancement-38960 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+New product block editor modal text enhancements
```
Comment (louwie17) [inline packages/js/customer-effort-score/changelog/enhancement-38960]: We should change this changelog to be CES specific.
In this case we are fixing the onsubmit label.

**Review by louwie17 — CHANGES_REQUESTED** [review]
Nice work on this @mdperez86, this tested well.
I left one comment in relation to the changelog for the CES package, if you could update that.
Also it looks like the Guide in the design uses a `primary` variant for the next button, I know this wasn't explicitly mentioned in the current issue.
So I can create a separate issue for it, but it should be a relatively simple fix if you could add it.
It just be matching the original Guide component in our copied version for this line: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/guide/index.tsx#L154
Here is our version: https://github.com/woocommerce/woocommerce/blob/e6eda7fce5796e8baf784f68787593a2ad62ed93/plugins/woocommerce-admin/client/products/tour/components/guide/index.tsx#L101

File: packages/js/customer-effort-score/changelog/enhancement-38960 (hunk 1 above)
Comment (mdperez86) [reply packages/js/customer-effort-score/changelog/enhancement-38960]: done -> [743725a](https://github.com/woocommerce/woocommerce/pull/39055/commits/743725aa8d48b12e218ddf67d808e7e816f32077)

**Review by louwie17 — APPROVED** [review]
LGTM 🚀 Thanks for the updates, this still tested well, thanks :)

**Comment by mattsherman** [comment]
@mdperez86 @louwie17 This PR broke the alignment of the feedback bar content... it used to line up with the form (same left/right margins). It now is offset...

Screenshot 2023-07-13 at 00 06 02

**Comment by mattsherman** [comment]
It also broke the positioning of the snackbar notices... they should appear above the feedback bar (they did). They no longer do -- they now overlap the feedback bar (they shouldn't).

Screenshot 2023-07-13 at 00 10 33

**Comment by mattsherman** [comment]
(I can create a new issue for these bugs later -- it's too late in the evening right now... just didn't want to lose track of them)

**Comment by louwie17** [comment]
> (I can create a new issue for these bugs later -- it's too late in the evening right now... just didn't want to lose track of them)

Good catch @mattsherman, I had not noticed those when reviewing. It be nice to get those in before the code freeze on Monday (if possible), especially the alignment one (cc @mdperez86 )

**Comment by mattsherman** [comment]
@louwie17 @mdperez86 Here is the issue for tracking the regressions...

https://github.com/woocommerce/woocommerce/issues/39228

**Pull request merged**